zoloto/marker.py

```diff
@@ -38,12 +38,14 @@ def _is_eager(self):
 
     @property
     def pixel_corners(self):
-        return [Coordinates(*coords) for coords in self.__pixel_corners]
+        return [Coordinates(x=x, y=y) for x, y in self.__pixel_corners]
```

Comment:
I don't believe this change is necessary?

Reply:
98ab842